pkg/scheduler/plugins/loadaware/helper.go

```diff
@@ -43,6 +47,96 @@ func getNodeMetricReportInterval(nodeMetric *slov1alpha1.NodeMetric) time.Durati
 	return time.Duration(*nodeMetric.Spec.CollectPolicy.ReportIntervalSeconds) * time.Second
 }
 
+func missedLatestUpdateTime(assignedTime, updateTime time.Time) bool {
+	return assignedTime.After(updateTime)
+}
+
+func stillInTheReportInterval(assignedTime, updateTime time.Time, reportInterval time.Duration) bool {
+	return assignedTime.Before(updateTime) && updateTime.Sub(assignedTime) < reportInterval
+}
+
+func getTargetAggregatedUsage(nodeMetric *slov1alpha1.NodeMetric, aggregatedDuration *metav1.Duration, aggregationType slov1alpha1.AggregationType) *slov1alpha1.ResourceMap {
+	if nodeMetric.Status.NodeMetric == nil || len(nodeMetric.Status.NodeMetric.AggregatedNodeUsages) == 0 {
+		return nil
+	}
+
+	if aggregatedDuration == nil || aggregatedDuration.Duration == 0 {
```

Comment:
what is the scenario where `Duration == nil` and the aggregated result of the max duration is picked?

Reply:
It's just a default strategy, and it's also an empirical strategy. Some users expect to schedule according to the data of the maximum aggregation period.

Reply:
The default strategy may be not obvious to understand. How about adding some comments?